Symfony/CS/Console/Output/ProcessOutput.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Console\Output;
```

Comment:
I'm not sure.. why shouldn't we want it? Im kind of used to namespace about everything so please explain :)